src/tags_int.cpp

```diff
@@ -596,6 +596,9 @@ namespace Exiv2 {
                 "is used instead of this tag. If this field does not exist, "
                 "the TIFF default of 1 (chunky) is assumed."),
                 ifd0Id, imgStruct, unsignedShort, 1, EXV_PRINT_TAG(exifPlanarConfiguration)},
+        {0x011d, "PageName", N_("Page Name"),
+                N_("The name of the page from which this image was scanned"),
```

Comment:
Missing end punctuation mark

Reply:
Thanks. The punctuation mark was missing also in the DocumentName... so I've added both of them now.